qiskit_experiments/framework/composite/parallel_experiment.py

```diff
@@ -92,12 +92,39 @@ def _combined_circuits(self, device_layout: bool) -> List[QuantumCircuit]:
             else:
                 num_qubits = 1 + max(self.physical_qubits)
 
-        joint_circuits = []
-        sub_qubits = 0
+        # Transpile component circuits
+        transpiled_circuits = []
+
+        # Max duration for all components that will be combined into a single circuit
+        max_durations = {}
+        duration_unit = None
+        scheduling_method = getattr(self.transpile_options, "scheduling_method", None)
```

Comment:
Will it be transpile option? I think we should keep current behavior of delegating this scheduling to sub experiments, and turn this into experiment option of parallel experiment. We should give it another name because the logic you implemented is too much simplified.

Reply:
We can still schedule individual experiment with different policy. Parallel one can prepend or append delay without modulating the individual schedule (the parallel scheduler considers the individual circuit as a block). I think this is another layer of scheduling, and thus we should avoid having the same option. Otherwise user may easily misunderstand the behavior of parallel block scheduling.